support-figma/extended-layout-plugin/src/shader.ts

```diff
@@ -131,3 +141,29 @@ export async function setShader(shader: string, shaderFallbackColor: string) {
     );
   }
 }
+
+////////////////// Clear //////////////////
+export async function clear() {
+  Utils.dcLog("Clear shader data from plugin data.");
+
+  await figma.loadAllPagesAsync();
+
+  for (let page of figma.root.children) {
+    for (let child of page.children) {
+      await clearNodeAsync(child);
+    }
+  }
+  figma.closePlugin("Shader data has been cleared.");
```

Comment:
This clears all shaders from the entire file, so maybe something like "All shaders have been removed".

Reply:
Done.